### PR TITLE
Github Actions Matrix Build: windows, mac, linux

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

**Comment by bjones86** [comment]
Cool!

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -9,26 +9,37 @@ on:
...
-          fetch-depth: 0
-      - name: Git Pull
-        run: |
-          GIT_MERGE_BASE_SHA=$(git merge-base origin/$GITHUB_BASE_REF $GITHUB_SHA)
-          echo "Commits before pull"
-          git log --oneline $GIT_MERGE_BASE_SHA~1..HEAD
-          echo "Pull with fast forward only"
-          git pull --ff-only origin $GITHUB_BASE_REF
-          echo "Commits after pull"
-          git log --oneline $GIT_MERGE_BASE_SHA~1..HEAD
```
Comment (ern) [inline .github/workflows/maven.yml:27]: This was put in to make sure fast forwards are working, why the removal?

Comment (ottenhoff) [reply .github/workflows/maven.yml:27]: It looks complicated! 

What does it do?!

**Review by ern — APPROVED** [review]

**Comment by jonespm** [comment]
It looks like JDK 11 is added but still failing. Seems like these should be removed or commented until it actually compiles in 11.

**Comment by ottenhoff** [comment]
JDK 11 support will show up very soon!

**Review by ern — CHANGES_REQUESTED** [review]
Not sure about adding windows as a build target.
Also would remove the jdk 8 build as master should only use jdk 11 other branches have their own maven.yml

**Pull request closed without merging**